Makefile

```
@@ -21,8 +21,9 @@ spec:
 	@echo "Generating spec..."
 	sh -c "cd spec && make build"
 
-.PHONY: generate
-generate: $(SCHEMAS_SOURCES) $(SCHEMAS_FINAL)
+.PHONY: schemas
+schemas: $(SCHEMAS_SOURCES) $(SCHEMAS_FINAL)
+	@echo "Generating schemas..."
```

the echo is executed after so better
```suggestion
	@echo "Generated schemas."
```

---

Done